include/command.h

```diff
@@ -88,6 +92,12 @@ class Command
         */
         ~Command();
 
+        /*!
+        *   \brief Run any type of Command on the RedisServer.
+        *   \param r A pointer to the RedisServer
+        */
+        virtual CommandReply runme(RedisServer * r);
```

Comment:
We usually do ``_`` for method names.  Should this be ``run_me``?  Also, we try to associated the ``*`` symbol with the type, so it should be ``RedisServer*``